# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: serkan@gentoo.org-20090319042831-azrs6hi2bhyf1mfi
# target_branch: ../mainline/
# testament_sha1: 69eb7f6ad524eab24851ad0e52b0d23a4b57daaa
# timestamp: 2009-03-19 06:33:22 +0200
# source_branch: ../i18n-javadoc.patch
# base_revision_id: andrew@operationaldynamics.com-20090308112053-\
#   v18zrak1zwyzkdyh
# 
# Begin patch
=== modified file 'build/faster'
--- build/faster	2008-12-20 12:47:05 +0000
+++ build/faster	2009-03-19 04:28:31 +0000
@@ -655,6 +655,7 @@
 	cmd += "-exclude org.freedesktop.bindings "
 	cmd += "src/bindings/org/freedesktop/bindings/Time.java "
 	cmd += "src/bindings/org/freedesktop/bindings/Version.java "
+	cmd += "src/bindings/org/freedesktop/bindings/Internationalization.java "
 	if not verbose:
 		cmd += " >/dev/null"
 

# Begin bundle
IyBCYXphYXIgcmV2aXNpb24gYnVuZGxlIHY0CiMKQlpoOTFBWSZTWYXy8nYAAd1fgCAwUWv//3/v
f/C////wUASekZKMVNV7sA4aaiBJ5NTyDUZkmTJoaDDSek0aPUaeUNqMgSkgp6jwynqZT1P0UPU9
Q0NGh6gBoA9QGjQOGmmRiMJpgIYBNMIwTEyGmRoaAZSjI0ZGgAaaAAANDCAAANBKEEAI00noaAIn
qaZTzSaan6p6jTJhBo2kmEdVkys2gm+j/gltnnP5zJIKjl4SNTLWkRCoexnkmzVFjywFxrzsCHiI
JodSjcy4AcJEGFblsaAuVGJ333qF8IVNLVuBHwQ65QsDTitU1ZbzDxyrjgN++iKktrQCuYFJwkVx
EDyTgm8/zTpJ+AKBbcBwNpfo8Lw8L5V0/ZQpn7RDEdBD5ydWqb3RUNlHQ1WUpEZfHJbBhgDdG91h
xM3QYAhFogsg5UUzAyCfgoIbSqk5SF+v15OfBkMSlMfFATdhcq2igwgyJEejyPkE9dSiSUY9XpVU
lP6EU4ExHEozPlRQZpbobAOIJnVgDnA0UjhT6kvMyawIV1vSKGBeBYYQJyzhUWmOsGBpQGAkCyrC
XMaaxi6k6thJIpMUgVtoH17VNZdIvwzk+SS1+uQW+DSUkSKwYLz81WFVjRiOPtsKRCmUmwnCywCr
LJlGSxUMz4LqnQ9Jygn0ZihCmIE6Q/BONoMoyUKEj222FAI4B85EuKcCJEuKTu30EeyzE8fUlhQm
sIVVkUI7ijkYlE/VT4dV1hKN2Y2UWdhpdXxM2BbKg2VLWEEgHXtjq06tgs6aH4RxglDLOK0B8IeL
vYrmqctuqZ3th2rW41M88PtPphMV2bfHYKWRn55YnFmuGg+aiBBqaS5Hd9w1J+wuAPY2hWubxNF3
XpuhVZ083bDNqpXnyKBQGix/esnnwghzP0Mgcz5c/mTdVNp422GM5oGQTkpKoLfRAxg2pEHUTXPS
jdgEB1P5NdqkAMqsiKvGCrHJtZEgD3eJL0DLwmD966CcoCY3EbIwqHHtu1TL/uMO5wXZ1+qW4E6V
OBiQR92I+hXOayVLnjBdtCtDdfmc3E1i8UUF2SGf1NMxYKia8hpa8qCBCOMkXojsKOx5VwSCHsRh
K6cbY8An8lEHnveHKTYTFRIuKlA1Chp3RDQotJQLClN548LSwaRgfGJdUryG5wXENAJlxomcQxVx
T709P8cjPv0wZCKWdgHJ/IDOF6KGz5doD7y2mBJfIUFsLNFYpCpnIXMZQ9H7CUNUREBCAUA5CAvo
0hk427F6QYgJ5abqIM4F1BO+M2hWwFMiPAESFSXxkylXtBUtindqmKwHhol73dZFnnJRGEOm783i
Go7QxGEs4ChHU7WqL/bjWKgj1E40uD1yiHsaBT3GdCQQN/EQgp4AwZ5noeoQy6nhV5RSxNM32mJt
msIEgE2+MRaiBovXrZHIZA6XIfH1UXHXOiVdpbYqO0hWCUyqiDAp0OtDEgPgoqneyNv1Ao2ACzCt
4OyHDR91tS3mjZXmXIRbclSiPrKgnYJphFkt9Npb2xIe803D3DVQMaSF9I4lAgIdhKgwkFC2Dhxh
sTWSU4R0BTZlqqSKORE+teoMnpATuhXtGZcOIHnSBmDxMA0L+UFrCEDb5w/xdyRThQkIXy8nYA==
